src/routes.js

```diff
@@ -7,7 +7,10 @@ import AiringTodayTv from "./components/tvShows/AiringTodayTv"
 import MostPopularTv from "./components/tvShows/MostPopularTv"
 import OnTv from "./components/tvShows/OnTv"
 import TopRatedTv from "./components/tvShows/TopRatedTv"
-
+import MovieDetails from "./components/MovieDetails.vue"
+import reviewsReadMore from "./components/reviewsReadMore.vue"
+import reviewsReadAll from "./components/reviewsReadAll.vue"
+import filmActors from "./components/filmActors.vue"
```

Comment:
Use PascalCase for component names for both imported component and file: e.g:
`import filmActors from "./components/filmActors.vue"`